cmd/car/car.go

```diff
@@ -34,6 +34,26 @@ func main1() int {
 				Usage:  "Detach an index to a detached file",
 				Action: DetachCar,
 			},
+			{
+				Name:    "extract",
+				Aliases: []string{"x"},
+				Usage:   "Extract the contents of a car",
```

Comment:
Looking at the code, it looks like if you give anything other than a car where the roots are DagPB/Raw, it will error. Perhaps it is worth putting a comment like this ("Extracts the contents of a car when the car encodes UnixFS data")

Reply:
We should think about other conventions that we should try to interpret, like hamts, structured data, etc.